version/version.go

```diff
@@ -1,8 +1,24 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package version
 
 var (
-	// ServingVersion is the version that will be applied to the KnativeServing resource.
-	ServingVersion = "0.14.0"
+	// ServingVersion is the default version that will be applied to the KnativeServing resource.
+	ServingVersion = "0.15.0"
 
 	// EventingVersion is the version that will be applied to the KnativeEventing resource.
 	EventingVersion = "0.14.2"
```

Comment:
Let's just remove this file. It's useless now that we're "multi-versioned" and they're determined by the embedded file structure, not this file.

Reply:
Done